src/solver/opamActionGraph.ml

```diff
@@ -82,8 +82,18 @@ module MakeAction (P: GenericPackage) : ACTION with type package = P.t
     | `Install p, `Install q
     | `Reinstall p, `Reinstall q
     | `Build p, `Build q
-    | `Fetch p, `Fetch q
+    | `Fetch [p], `Fetch [q]
       -> P.compare p q
```

Comment:
This case can be removed ?

Reply:
it can yes... i kept it to not define/cal la function in case of single package, which is most of the cases. But maybe non sens  optim